web_scraper.py.py

```diff
@@ -0,0 +1,63 @@
+from bs4 import BeautifulSoup
+import requests
+
+def parse_pet(href):
+    html = requests.get(href)
+    html_text = html.text
+    soup = BeautifulSoup(html_text, 'lxml')
+    details = soup.find('ul', class_ = 'petdetails')
+    indents = details.find_all('li')
+    #W poniższych zmiennych zapisze dane, -1 oznacza brak danych
+    age = -1
+    breed = -1
+    gender = -1
+    weight = -1
+    number = -1
+    status = -1
+    date_in = -1
+    date_out = -1
+    box = -1
+    address_found = -1
+    group_link = -1
+    group_name = -1
+    for indent in indents:
+        if (indent.text.startswith("W typie rasy")):
+            breed = indent.find('strong').text

```

Comment:
Należy upewnić się co się stanie jeżeli brakuje danych na stronie. Czy będzie błąd czy wartość zwrócona to null, czy jeszcze coś innego.